src/app/pages/network/forms/interfaces-form.component.ts

```diff
@@ -561,9 +571,22 @@ export class InterfacesFormComponent extends ViewControllerComponent implements
         this.entityForm.setDisabled('lacpdu_rate', true, true);
         this.entityForm.setDisabled('xmit_hash_policy', true, true);
       }
+      if (value === LinkAggregationProtocol.Failover) {
+        this.lagPortsField.tooltip = helptext.lagg_interfaces_tooltip + ' ' + helptext.lagg_interfaces_failover_tooltip;
```

Comment:
The tooltip should be one string which properly describes the special case for `Failover` link aggregation protocol at the end. So, I think it's better to make an addition to `helptext.lagg_interfaces_tooltip` instead of having another string that's added conditionally. This looks like someone moving code around could easily forget to add the extra bit and we'd lose important information that's very relevant to this input.